docs/docs/dynamo-store/access-strategies.md

```diff
@@ -70,3 +71,63 @@ With this in place Equinox will maintain and store a Snapshot event in the tip
 document that is guaranteed to always be up to date, and as a result the state
 of the Stream can be reconstructed with a single point-read.
 
+## RollingState(toSnapshot)
+
+`RollingState` will throw away events, only storing the snapshot. This can be
```

Comment:
```suggestion
`RollingState` builds state from events, just like the other modes. The key difference is that it discards the events, only storing the snapshot. This can be
```